api/helpers/gamestate/moves/resolve/two.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+  friendlyName: 'Resolve Two One-Off',
+
+  description: 'Returns new GameState resulting from resolving a two, which scraps target royal and glasses eight on the field, and counter one-off cards.',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the two resolves',
+      required: true,
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move.',
+    }
+  },
+
+  sync: true, // synchronous helper
+  fn: ({ currentState, playedBy }, exits) => {
+    let result = _.cloneDeep(currentState);
+    result.targetCard = result.oneOffTarget;
```

Comment:
I added this to ensure that the original oneOffTarget (e.g. the opponent king) is correctly set as the `targetCard`, as opposed to the target from the previous move, which could have been a counter. This lets the new test I added pass